<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Picker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="first_layout">
      <item>
       <widget class="QLabel" name="shdtype_label">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Shader Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="shadetype_box"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="manual_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="second_layout">
      <item>
       <widget class="QGroupBox" name="first_groupbox">
        <property name="title">
         <string>Basic</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5"/>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="second_groupbox">
        <property name="title">
         <string>Transparency</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2"/>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="third_groupbox">
        <property name="title">
         <string>utilites</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="return_prev_btn">
           <property name="text">
            <string>Reassign Preview Shade</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="del_unused_nodes_btn">
           <property name="text">
            <string>Delete Unused Nodes</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
